.github/workflows/android-maestro.yml

```
@@ -87,32 +90,48 @@ jobs:
         steps:
             -   name: Checkout repository
                 uses: actions/checkout@v5
+                timeout-minutes: 10
 
             -   name: Setup Node
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)